### PR TITLE
Daily updates

### PR DESCRIPTION
**Pull request by amit-ib**
Added Google Login Authentication

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,69 @@
...
 import "./assets/scss/styles.scss";
 import TodoForm from "./components/todo/TodoForm";
 import TodoList from "./components/todo/TodoList";
 import { TodoModal } from "./models";
+import { UserModal } from "./models/user";
 import { staticTodo } from "./mock-data/todo";
+import Login from "./components/shared/Login";
+import { gapi } from "gapi-script";
+
+const clientId =
```
Comment (akash-dhande-ib) [inline src/App.tsx:11]: Keep the sensitive information in env file, out of repository.

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,69 @@
...
+
   const [todos, setTodos] = useState<TodoModal[]>(staticTodo);
+  const [userProfile, setuserProfile] = useState<UserModal | null>();
+
   return (
-    <div className="container">
-      <h3 className="text-center">Todo List</h3>
-      <TodoForm todos={todos} setTodos={setTodos} />
-      <TodoList todos={todos} setTodos={setTodos} />
+    <div>
```
Comment (akash-dhande-ib) [inline src/App.tsx:36]: Avoid unidentified tags, ie. empty tags.

File: src/assets/scss/_common.scss [hunk 3]
```diff
@@ -1,11 +1,49 @@
...
+}
+
+.welcome-text {
+  color: $white-color;
+  padding-top: 20px;
+  text-align: center;
+  .button-login {
+    text-align: left;
+    display: inline-block;
+    button {
```
Comment (akash-dhande-ib) [inline src/assets/scss/_common.scss:35]: I guess it's not common styles.

File: src/components/shared/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { GoogleLogin, GoogleLogout } from "react-google-login";
+import { useState, useEffect } from "react";
+
+const clientId =
```
Comment (akash-dhande-ib) [inline src/components/shared/Login.jsx:4]: Keep it in env file

**Review by akash-dhande-ib — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (amit-ib) [reply src/App.tsx]: .env file created and placed clientId there

File: src/App.tsx (hunk 2 above)
Comment (amit-ib) [reply src/App.tsx]: removed empty div

File: src/components/shared/Login.jsx (hunk 4 above)
Comment (amit-ib) [reply src/components/shared/Login.jsx]: .env file created and placed clientId there

**Review by akash-dhande-ib — APPROVED** [review]

**Pull request merged**